src/prefab.ts

```diff
@@ -5,12 +5,15 @@ import Identity from './identity';
 import Loader, {LoaderParams} from './loader';
 import {shouldLog} from './logger';
 
+type EvaluationCallback = (key: string, value: ConfigValue, context: Context | undefined) => void;
+
 type InitParams = {
   apiKey: string;
   identity?: Identity;
   context?: Context;
   endpoints?: string[] | undefined;
   timeout?: number;
+  trackEvaluation?: EvaluationCallback;
```

Comment:
I don't have strong feelings about naming this, LMK if you have a better idea

Reply:
🤔 `trackEvaluation` feels perhaps a little too prescriptive.

How do you feel about `afterEvaluationCallback` ?

Reply:
Works for me!